overlay/3-node/patches/pullsecret-sealed-patch.yaml

```diff
@@ -0,0 +1,15 @@
cyAyLvMjcT+9H0k+T6JHxlSZrXe4giuqOwKinkzgkDMiNfWckpTS3tWkOruPclN3wjcsYr8+5VCxk14CjUjR+F/YfRzWTNmceqrJR8fy2F8VxmJR53b0TxzXS2WTDx6tAZxVIqcNKZ6kItTxd1bbG++VBENfliLE1OTeAVVle0xgqNR05vlNSTPNlIfUGMh9znUYb5JjkIOr2dqs34Ol7+tAFQYS7DMvY6EbB3D8RdOdiF43tcYH1827ni2nY8hxoziCfGZRugaTUuLsYg2FMmkEwn8zT67vznsBf0J0ja6bUrGan7AXcj/2OUiv/Ztmzt0ZF4DUWK5dCt4IIMEVFreVcvcQEMwe21wgWgiCOYRkuJfff0zEy90ChOSXKB7AZ7kv5cwN+kf+LP/Xr/CAl5dubY4L03fJ+X5/FUAEPDWoR79iNgIlnGwzehftK1iY/ImR7B8H3lmsFciiw7Fruszx8yEk+5dQEOjC3xMhZgL87mAIZu35lv1ykt1cVNI1T/w8UUVxELd8zUHjbRAKXabNJyso/goKkZWCycqc7WA0Fua1tWboD/kENF4IVkK4iOIprDM9fkfkaSjOgYcE1SDJMNwFXcAPBg2K9cLgkgsgW6H1P/vM5dGn/V3cBuBCLs8a3DcLzie85vYqxJvzjedp/pngza8A4ehn0gMFYtzyD+xnvX6lvxD73AzspVtY6gxt5TpXCTwC+4j1tk/utuEXHjJtPJPvFwLxHFEF/j1mjoo35PUY9jvoy+uaHCDBCayUT29x0vDUnF/DvrPJM3Dcv6W8LwR8lTWZEnEfBPd6UA8msJMzqN0cC9frmfkrdsYEIWfEz2t9EOLZFxZtlkAeP49q9RBF4Lfs28Kw/5fFfFeZLtvWCMzW3I5ru9Xuzc7es9brdN9nCEuwqdR9TFQBxrr14xBqpOsw0c6gpIEKHO3WA3gsfqNOXXHjG7oofae0diqz502zmhyIy3+D3jnnUJKVWbMONa2rDFnMEGCEPHp6vQ1BRxjjdaxW9ySnwiDeg0UoIcHhteZGGjuewh3+nkkY6oWFkI6AYKfbBXDFJ3oqBci8xFf6o4U1Vj/PGW+pelinbiTWOp3Ev6oDM4GE6U8OEs5W91sd5b6YQJS8/tDu7tr/SxQKv1SsscKZAeX5cucKGhaaxL/QYVbVNUfuTPNdjT6zAAqoEh5pjHNjAaTFrwEGqn/z1CraNA5rk+KWYWe6bfS9BoytZQrBEZvX9q00SAToSx/j2CGEolTwP1IqLRzH3wyl9WdvSwQ5hArB6PUCAlRxUaFOSd3PZdLNb2UrzHgGGsilljNPQJIQroytHjQzVc9lSRHrfv4SQ8ps6ALcbMYLQKMoo2i9HI8EksaZx3LLzB6FTTBb+lP/NW+Jndod4ZGtrELhXTmGpd/Wq2yG316PYdVsZy20amUjzNrngI8XZxCn5fdrPUzJs7O/vP6sWCNCfrzkwkCqMB7JKtm/9TTDzqnDBeB5rcsaRvdwqPIis3vMEuCHmh4/T5KHIycYLiSu3hL4gvWF8q2dwQaE1RYM4FOzWdfs2gmGPQ3AxOZESpwYd/y338hjKnf2AZsK52SOSPgjTPu8TOehkP2NmjuhZHJZyDYpP3exz+nYXNYyEdarfGR0GCLjtE9pBsUKjf0XcHT+xFRK3QaqLl+EZQrvPP7Uu3a+48RH9hEOdT6CBsoynMnlG2xJfeAZMfqpCHgueOtJbtqs6cQjE/IXHlY48OGU/8Cyb0UbAPUOIP6ujazI4PMotVF1cQnjM0lzkEo19BXqQlfKtbkj74fkTusDXr6m8eF3tzjdlHhcqZA+/gb9cc9D1HGW1CFXE9IdAcC08eLaexYlalgUlddcEuUNytx67R1rtz6f7CwKUX2YQZ3yKtXX0KrNJi1zgaLfhDeawIChzzOsF9+vLMxHwVPuXBP1icb/5tR4pOPrH5W9bcwmx3dscD2tYyOGwCBNjgLv0LvAUNjtYJK/InOm2whKlQem72+xSvpFHpHGEEcwkEUoYxUCOz+NoBccBsbKlT3cN8xIgMlblIJCHFEEmgPklvCdVSPDTyTlK+FWQCB/cV6xLfus9H412lovhoxmCTKjocRTbitp1cDgBRywihFAuOOecOAhbHJKSK1NWT3O8EJyF2mTd3ObzvOtkI1GpqawgCfsN77ZJiusNwphE0EuwQfUvSzE2mhvkAvZ2ISCiwVt3s2l98khkBwniMx5OtuzCVa+xu5AdDJWEMdQyc+2REnzvso9SvTloVO5K7YFpsm6XSnS6lbRyvjj9YTlqwbe8qXRCFpdGO0jpu9iZF1uiuryk4WVRlknTOcXvatYvc7SE8NrmZIxoqXMCRth1bSwrdpw90dt9oMdn3tgDtfMmlA7J96FFz/ni9Ph+j0zZdGAC3Thu7FC2IcJQMpTrdP/7uZ9c2bM7hlOeFZCj/GhSus8YYUKOq0iNCe/pOYiIK2DwZg9sCchXF9ZRxJ5KLWMkp3q9lOqJbX/qezmSLNK9BTJFWVVQHydJHKrEeJENkjZPQ3u51YRIt25Mi7n1svpzVr98sxwzBfnrGWTtSSOjNvdQXEzj1yfnau8yKPEhOSUeWpDZ2qclxfE4Du962MXxjympRTjeeDapbITN9E+IvsVjEO5zbvadES81PfaPqwE509FZOpKhM02zYZuqBZS3fBpKrYTSLjaJZNotxunRXfky4UVXRnq60Hork3XNZPDyx1sJP3XY7g+on5+UZMGgOLvTaUgbguMHDpT1kwI3T4jVVOe9IyJyQ7hObsiOY1YbY4BLCu4T2BFCVmkQntN6B7FM1JQxcQ5m0Og4mJmU0cZJ+9xCu5+fdSk4yFFIbrjZ5g3kq5OJmHj84aN28Gi/JMUaNR9+5Lo9a4arYYzRO6Y19orsRGSXOTrRCUMYivjNeU6QXguoeQXd7N5W1neYPjCAd+4Tngd85JFh4YZQ29fhotIBtE1H3WsWEUFwMizjzBQLlzIts3CB0sF13oz
```

Comment:
Maybe this value should be in a string? there are a lot of special characters in the encrypted secret

Reply:
done